Formula/portable-ruby.rb

```diff
@@ -33,18 +67,18 @@ def install
       --prefix=#{prefix}
       --enable-load-relative
       --with-static-linked-ext
-      --disable-dln
```

Comment:
Nice! It's still statically linked with this?

Reply:
Yes and with it specified you cannot load any native extensions at runtime 😨